exporter/azuremonitorexporter/azuremonitor_exporter.go

```diff
@@ -33,6 +34,10 @@ func (exporter *azureMonitorExporter) Start(_ context.Context, _ component.Host)
 	exporter.config.InstrumentationKey = configopaque.String(connectionVars.InstrumentationKey)
 	exporter.config.Endpoint = connectionVars.IngestionURL
 	telemetryConfiguration := appinsights.NewTelemetryConfiguration(connectionVars.InstrumentationKey)
+	telemetryConfiguration.Client, err = exporter.config.ClientConfig.ToClient(ctx, host, exporter.settings)
```

Comment:
Will it break existing functionality when user not specifying `auth` setting?

Reply:
Since it is not a pointer but rather a struct, it will just end up using a default http client so it shouldn't break anything in theory. In my test I didn't even set the endpoint value so it was technically using a default http client except for the authenticator.